ios/CakeWallet/wakeLock.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  wakeLock.swift
+//  Runner
+//
+//  Created by Godwin Asuquo on 1/21/22.
+//
+
+import Foundation
+import UIKit
+
+func enableWakeScreen() -> String {
+    UIApplication.shared.isIdleTimerDisabled = true
+
+    return "screen wake turned ON"
+}
+
+func disableWakeScreen() -> String{
+    UIApplication.shared.isIdleTimerDisabled = false
+    return "screen wake turned OFF"
+}
```

Comment:
Same here. If you need to pass some argument into result function, better pass nil(null) in case if you expect void as result for the function, or pass boolean value based on state what changed your function.